praw/models/reddit/mixins/inboxtoggleable.py

```diff
@@ -28,7 +28,7 @@ def disable_inbox_replies(self):
         )
 
     def enable_inbox_replies(self):
-        """Enable inbox replies for the item.
+        """Enable inbox replies for the item. This can only apply to a thing created by the user.
```

Comment:
```suggestion
        """Enable inbox replies for the item.
        
        .. note::
        
            This can only apply to items created by the authenticated user.
```

Like I mentioned above, let's add a note like this instead.